json-endpoints/jetpack/class.jetpack-json-api-user-create-endpoint.php

```diff
@@ -26,16 +26,19 @@ function validate_input( $object ) {
 
 	function create_or_get_user() {
 		require_once JETPACK__PLUGIN_DIR . 'modules/sso/class.jetpack-sso-helpers.php';
-
 		// Check for an existing user
 		$user = get_user_by( 'email', $this->user_data['email'] );
-
+		/**
+		 * Fires after a user accepts an invitation and before a user gets created.
+		 * @return WP_User|false WP_User object on success, false on failure.
```

Comment:
Could you add the `@module sync` here as well, as well as the `@param` and `@since` blocks?

For reference, each hook should always include those elements, in that order, with blank lines between them:

```php
/**
 * Description.
 *
 * @module sync
 *
 * @since 5.8.0
 *
 * @param type $var Description.
 */
```

Thank you!

Reply:
Thanks for the review 👍 . Will update it shortly